acceptance-tests/xfcc_test.go

```diff
@@ -293,3 +351,32 @@ func parseXFCCHeader(header string) string {
 
 	return certPEM.String()
 }
+
+func parseXFCCHeaderToCert(header string) *x509.Certificate {
```

Comment:
`parseXFCCHeader` should probably updated to call this new method, rather than duplicate the parsing (and maybe renamed to something more specific, like `parseXFCCHeaderToPEM`)

```
func parseXFCCHeader(header string) string {
	cert = parseXFCCHeaderToCert(header)

	var certPEM bytes.Buffer
	err = pem.Encode(&certPEM, &pem.Block{Type: "CERTIFICATE", Bytes: cert.Raw})
	Expect(err).NotTo(HaveOccurred())

	return certPEM.String()
}
```

Reply:
good idea.